src/MSAL.Common/TokenCacheExtensions.cs

```diff
@@ -44,27 +46,52 @@ public static void SetBeforeWrite(this TokenCache tokencache, TokenCache.TokenCa
         /// <summary>
         /// 
         /// </summary>
-        /// <param name="tokencache"></param>
+        /// <param name="tokenCache"></param>
```

Comment:
If you feel the summary / params are self-explanatory, it would be better to just delete them.